src/picongpu/include/particles/manipulators/ProtonTimesWeightingImpl.def

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 Axel Huebl
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+namespace picongpu
+{
+namespace particles
+{
+namespace manipulators
+{
+
+/* Re-scale the weighting of a cloned species by numberOfProtons
+ *
+ * When cloning species from each other, the new
```

Comment:
Here, you are speaking about a general `species` but this manipulator is only useful for electrons. Could you specify `species` by saying `electron species` here (as in the description of the pull request) in order to clarify the comment?